dojson/contrib/marc21/utils.py

```diff
@@ -126,7 +126,22 @@ def split_blob(blob):
     raise StopIteration()
 
 
+def split_stream(stream):
+    """Split the stream using <record.*?>.*?</record> as pattern."""
+    buf = []
```

Comment:
@jirikuncar I cannot find a way in lxml to obtain back the exact inner string. Can you?